### ISSUE TITLE
Fraction.GCD() doesn't handle NaN

### ISSUE THREAD

**Issue by jeffcann** [0xfe/vexflow#816]
This is a bit of an edge case and low priority, but the Fraction.GCD calculation doesn't handle when the second arg is NaN.
https://github.com/0xfe/vexflow/blob/00c8a0d17092c24e40dd4519d4b8ec59c711923c/src/fraction.js#L17

I discovered this today when I set the timing signature to '0/0', it will go in to an infinite loop and cause the Chrome tab to hang.

Would be best if this could throw an exception, perhaps something like:
```
if ((typeof a !== 'number' || Number.isNaN(a)) || (typeof b !== 'number' || Number.isNaN(b))) {
  throw new Vex.RERR('BadArgument', `Invalid numbers: ${a}, ${b}`);
}
```

For context, I'm building an exercise where the user will listen to a piece of music and has to input the timing signature. I was using '0/0' as the initial timing signature.

**Comment by 0xfe** [comment]
Thanks, yes. If it causes the tab to hang, then an exception is best here. Make sure that the all tests still pass though. :-)

(Also, we should reject zeros.)

**Comment by rvilarl** [comment]
@0xfe you can close this one.

**Comment by rvilarl** [comment]
@0xfe already implemented in #1071
